shared/actions/login/index.js

```diff
@@ -420,6 +432,16 @@ export function addANewDevice () : AsyncAction {
     })
 
     const incomingMap = makeKex2IncomingMap(dispatch, getState)
+    incomingMap['keybase.1.provisionUi.chooseDeviceType'] = ({sessionID}, response) => {
```

Comment:
instead of showing the screen asking which type, we already made that choice so we overload the response here. maybe  a useful pattern. let's discuss